doc/help/embed.md

```diff
@@ -0,0 +1,271 @@
+
+# Login
+
+![Login](login.png)
+
+# User Management
+
+Click on the User Management Tab to see the User Management
+screen. This allows you to search for any user in the system by their
+registered email address and add them to any programmes or projects
+that you are a Programme Manager or Project Manager for.
+
+![User Management](user-management.png)
+
+## Editing Users
+
+If you are an administrator you can set the overall role for a user as
+User, Admin or Super Admin. If you are not an administrator but are a
+Programme or Project Manager, then you can make a user into a
+Programme Manager, Project Manager or user on any Programme or Project
+you manage.
+
+All managers and administrators have rights to edit data in programmes
+or projects they have been added to. These rights cascade down into
+the projects, properties, devices etc contained in the Programme or
+Project they are a manager for.
+
+All users have the right to view private data in the programmes or
+projects they have been added to.
+
+![Editing User](editing-user.png)
+
+Here is an example of making a user a Programme Manager.
+
+![Make user a programme manager](user-as-programme-manager.png)
+
+This is what it looks like when you have successfully edited a user.
+
+![Successful Editing](successful-user-editing.png)
+
+# Programmes
+
+Programmes allow you to have multiple projects in with one overview
+and leadership team. Project Managers and users on projects can only
+see data in their projects allowing for privacy between project teams,
+but an overview for the programme team.
+
+## Before Adding
+
+![Programmes Listing](programmes-listing.png)
+
+## Adding
+
+![Editing Programmes](edit-programmes.png)
+
+## After Adding
+
+![After adding programme](after-adding-programme.png)
+
+## Viewing
+
+![Programme Details](programme-detail.png)
+
+# Projects
+
+Projects are a way of getting people together to work on a group of
+properties. All data in a project are editable by a Programme or
+Project Manager. Users in a project can view all private data in that
+project.
+
+## Adding a Project
+
+Projects must have a project name.
+
+![Adding a project](adding-project.png)
+
+## Listing Projects
+
+![Listing projects](projects-listing.png)
+
+## Viewing project details
+
+![Project details](project-details.png)
+
+# Uploading Property Details
+
+You can upload a csv file to create a property with profile
+details.
+
+![Property upload](add-properties.png)
+
+To make it easier to create the csv an Excel template has been
+created. To use it, fill in the Excel spreadsheet as below and save
+the file as csv when you are done. The csv can be uploaded as in the
+picture above.
+
+## Property Details Form
+
+[This Excel spreadsheet](example-property.xlsx) has been designed to
+comprehensively capture the diverse range of building characteristics
+(house type, construction, technologies installed); plus very
+top-level info on the initial design intention and final output. The
```

Comment:
I'd say "overview" or "general" rather than "top-level" ("top-level" to me could imply "very important")